robottelo/hosts.py

```diff
@@ -1273,6 +1385,17 @@ def capsule_setup(self, sat_host=None, **installer_kwargs):
                 f'A core service is not running at capsule host\n{result.stdout}'
             )
 
+    def enable_mqtt(self):
+        result = self.execute(
+            'satellite-installer \
```

Comment:
Its good practice to use the `InstallerCommand` class to construct these.

https://github.com/SatelliteQE/robottelo/blob/master/robottelo/helpers.py#L345

Reply:
I don't get how it helps, I see sat-installer being called directly more often then not around robottelo. If you could suggest how to rewrite that bit the InstallerCommand way, I'd be grateful

Reply:
https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/component/puppet.py#L9-L37

Reply:
@Gauravtalreja1 thanks, that's a great example!

@pondrejk the InstallerCommand class was introduced without converting every use of `satellite-installer` to use the class. We're encouraging people to use it as much as possible though, eventually converting all the uses of it.

I'm not going to call it required for this test, but I'd encourage it if you have a bit of time to make this switch, which should be straightforward in this case.

Reply:
Thanks for pointers, the function is now using InstallerCommand